README.md

```diff
@@ -1,7 +1,7 @@
 # Djinni Generator
 
-![Build](https://github.com/cross-language-cpp/djinni-generator/workflows/CI/badge.svg)
-![GitHub release (latest by date)](https://img.shields.io/github/v/release/cross-language-cpp/djinni-generator)
+[![CI](https://github.com/cross-language-cpp/djinni-generator/actions/workflows/main.yaml/badge.svg)](https://github.com/cross-language-cpp/djinni-generator/actions/workflows/main.yaml)
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/cross-language-cpp/djinni-generator)](https://github.com/cross-language-cpp/djinni-generator/releases/latest)
```

Comment:
If you click on the badge you are now directed to the latest release. I thinks thats nice. :P